app/assets/javascripts/components/accordion.js

```diff
@@ -25,20 +23,20 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
       $(window).unload(storeScrollPosition);
 
       // Indicate that js has worked
-      $element.addClass('js-accordion-with-descriptions');
+      $element.addClass('app-c-accordion--active');
 
       // Prevent FOUC, remove class hiding content
       $element.removeClass('js-hidden');
 
-      var $subsections = $element.find('.js-subsection');
-      var $subsectionHeaders = $element.find('.subsection-header');
-      var totalSubsections = $element.find('.subsection-content').length;
+      var $subsections = $element.find('.js-section');
```

Comment:
Huge fan of renaming subsection -> section, but it would've been nice to have renamed the variables too.

Reply:
That's a good point. Possible laziness on my part.